samples/bot-types-of-cards/nodejs/bots/botsuggestedCards.js

```diff
@@ -0,0 +1,213 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const { ActivityHandler, MessageFactory, CardFactory } = require('botbuilder');
+const { ActionTypes } = require('botframework-schema');
+const AdaptiveCard = require('../resources/adaptiveCard.json');
+const Office365ConnectorCard = require('../resources/o365ConnectorCard.json');
+const ThumbnailCard = require('../resources/thumbnailCard.json');
+const ReceiptCard = require('../resources/receiptCard.json');
+const ListCard = require('../resources/listCard.json');
+const CollectionCard = require('../resources/collectionsCard.json');
+
+class BotSuggestedCards extends ActivityHandler {
+    constructor() {
+        super();
+
+        this.onMembersAdded(async (context, next) => {
+            //Send welcome message when app installed
+            await this.sendWelcomeMessage(context);
+
+            // By calling next() you ensure that the next BotHandler is run.
+            await next();
+        });
+
+        this.onMessage(async (context, next) => {
+            const text = context.activity.text;
+
+            // Create an array with the valid card options.
+            const suggestedCards = ['AdaptiveCard', 'HeroCard', 'ListCard', 'Office365', 'CollectionCard', 'SignIn', 'ThumbnailCard'];
+
+            // If the `text` is in the Array, a valid color was selected and send agreement.
+            if (suggestedCards.includes(text)) {
+
+                switch (text) {
+                    case "AdaptiveCard":
+                        await context.sendActivity({ attachments: [this.sendAdaptiveCard()] });
+                        break;
+
+                    case "HeroCard":
+                        await context.sendActivity({ attachments: [this.sendHeroCard()] });
+                        break;
+
+                    case "Office365":
+                        await context.sendActivity({ attachments: [this.sendOffice356Card()] });
+                        break;
+
+                    case "ReceiptCard":
+                        await context.sendActivity({ attachments: [ReceiptCard] });
```

Comment:
I can't see this card. Why did we add it.?

Reply:
fixed